moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -718,6 +728,16 @@ bool ompl_interface::ModelBasedPlanningContext::solve(double timeout, unsigned i
   return result;
 }
 
+ompl::base::PlannerTerminationCondition ompl_interface::ModelBasedPlanningContext::getPlannerTerminationCondition(
+    double timeout, const std::shared_ptr<ob::PlannerTerminationCondition>& ptc_or)
```

Comment:
This doesn't use any member variables, can it not be a class method? Also unclear to me where this would be used anyway, it's not significant enough to be a separate method IMO.

Reply:
It could, I just thought it might make sense to move this to an extra method to add some parameter processing